src/runrms/config/_rms_config.py

```diff
@@ -134,13 +132,8 @@ def _set_config_file(self, config_path: str | None) -> str:
         config_file = DEFAULT_CONFIG_FILE
 
         entry_points = importlib_metadata.entry_points()
-        if sys.version_info >= (3, 10):
-            # Python 3.12 does not implement __iter__ on this object.
-            selections = entry_points.select(group="runrms", name="config_path")
-        else:
-            selections = [
-                ep for ep in entry_points.get("runrms", []) if ep.name == "config_path"
-            ]
```

Comment:
This package has been >= Python 3.11 so this condition is not longer needed (flagged by lint)